src/benchmark.cc

```diff
@@ -411,6 +415,8 @@ bool BenchmarkFamilies::FindBenchmarks(
         instance.min_time = family->min_time_;
         instance.use_real_time = family->use_real_time_;
         instance.use_manual_time = family->use_manual_time_;
+        instance.last_benchmark_instance = (args == family->args_.back());
```

Comment:
there's a possible bug here: you'll set this to true, but then it can be filtered out by the regex match below (l444). you only want to set it if the args is the last one and if it's going to be added to the benchmarks set.